src/containers/BooksList.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+import styles from '../css/modules/BooksList.module.css';
+
+const BookList = ({ books }) => {
```

Comment:
you'll need to hold state for the form, this should have been a class component.

Reply:
Yes, I did that in the next milestone #3 . I think using a functional component for a stateless component is better. Also, my eslint configuration won't allow it.